FlowCrypt/Models/Realm Models/KeyInfoRealmObject.swift

```diff
@@ -70,7 +70,8 @@ extension KeyInfoRealmObject {
 
 extension KeyInfoRealmObject {
     /// associated user email
-    var account: String? {
-        user?.email
+    var account: String {
+        guard let email = user?.email else { fatalError() }
```

Comment:
I'm not sure it is good to use `throws` here because having such error already makes app wrong. So it is better to guaranty that we always have `user` property when we create/update object.

Reply:
Sure, assuming our own code in the future will never be buggy. Would you bet your neck on that? I wouldn't :-)

Reply:
Still prefer throws here rather than `fatalError`.

Reply:
It makes code hard to use. We use this property in select queries, and having all such method throws will make code verbose.

Reply:
I'd still like to look into this in the future. Will file another issue.